addons/spreadsheet_dashboard/security/security.xml

```diff
@@ -5,5 +5,16 @@
         <field name="groups" eval="[(4, ref('base.group_user'))]"/>
         <field name="domain_force">[('group_ids', 'in', user.groups_id.ids)]</field>
     </record>
+
+    <record model="ir.module.category" id="dashboard_management">
+        <field name="name">Dashboard</field>
+        <field name="description">User access level for Dashboard module</field>
+        <field name="sequence">30</field>
+    </record>
+
+    <record id="dashboard_manager" model="res.groups">
```

Comment:
according to [guidelines](https://www.odoo.com/documentation/16.0/contributing/development/coding_guidelines.html#security-view-and-action)
```suggestion
    <record id="spreadsheet_dashboard.group_dashboard_manager" model="res.groups">
```